server/lib/mappings/result-data-sample.json

```diff
@@ -73,7 +73,12 @@
                     "type": "long"
                 },
                 "measurement_title": {
-                    "type": "text"
+                    "type": "text",
+                    "fields": {
+                        "measurement_title_kw": {
```

Comment:
Doesn't this make it so that the reference is `...measurement_title.measurement_title_kw`?  If so, could we consider something like:

```
"measurement_title": {
    "type": "keyword",
    "fields": {
        "raw": {
            "type": "text"
        }
    }
}
```

Reply:
It actually does; my brain apparently atrophied during the break, because the usage I _remembered_ seeing when I researched this way back last year was that ES didn't scope the name. I can vividly recall an example. But I can't find it now. The subfield name __is__ scoped, and thus you're right that simply naming it `"raw"` makes more sense.